guardian/manage-frontend
# DigiSub routes and cancellation reasons

## What does this change?
Adds in the routes for the new Digi Sub saves journey.

Adds in new screen with the DigiSub Cancellation reasons



## How to test

Go to `/digital/?withFeature=digisubSave` and see new Manage product page. Click 'Cancel subscription'. Follow cancellation (except Discount). See Reasons at end of journey.



## How can we measure success?



## Have we considered potential risks?



## Images
New cancellation reasons screen 
<img width="865" alt="image" src="https://github.com/guardian/manage-frontend/assets/115992455/180d9713-bbf0-469e-bfb6-5f7fd667d9b9">



## Accessibility
